tests/modules/segmentation/qc_utils/test_segmentation_video_utils.py

```diff
@@ -44,15 +61,36 @@ def example_rgb_video():
     return data
 
 
-@pytest.fixture
+@pytest.fixture(scope='session')
 def example_unnormalized_rgb_video():
     rng = np.random.RandomState(6125321)
     data = rng.randint(0, 700, (100, 60, 60, 3))
     return data
 
 
-@pytest.fixture
-def chunked_video_path(tmpdir):
+@pytest.fixture(scope='session')
+def example_unnormalized_rgb_video_path(
+        tmpdir_factory,
+        example_unnormalized_rgb_video):
+    tmpdir = pathlib.Path(tmpdir_factory.mktemp('eg_unnorm_rgb_video'))
+    # write video to a tempfile
+    h5_fname = tempfile.mkstemp(dir=tmpdir,
```

Comment:
I think this would just work:
```
h5_fname = tmpdir / "name.h5"
```
there shouldn't be a need to invke `tempfile` as you're already inside of a tmpdir.

Reply:
and the file would get removed with your `rmdir`

Reply:
actually, I guess not, it has to be empty.